requirements.txt

```diff
@@ -7,3 +7,4 @@ PySide6_Addons>=6.5.1.1,<=6.6.3.1
 PySide6_Essentials>=6.5.1.1,<=6.6.3.1
 typing_extensions>=3.10.0.0,<=4.11.0
 ujson>=5.8.0,<=5.9.0
+parametrized==0.9.0
```

Comment:
this shouldnt be a _production_ depedency, I'd suggest to create `requirements-dev.txt` or similar and put it there instead.

Reply:
That's a good point. By the way, do you guys manually add those dependencies to the requirements.txt or can pip do that automtically like npm can?

Reply:
There's `pip freeze` which lists all currently installed packages, so you can do `pip freeze > requirements.txt` but it lists even the transitional dependencies which is suboptimal.

Ideally the project should use some advanced dep. management tool like Poetry or pdm, but seems like that isnt/wasnt a priority yet

Reply:
Fair point, 
I don't think there's currently a plan to switchover to a more robust dependency manager, not saying a switch shouldn't happen but I believe the energy is more focused on swapping the backend from a JSON file to a database. 

Are there any recommendations aside from something like a requirements-dev.txt file to be a stopgap on this kind of thing?

Example
```
-r requirements.txt
parameterized==0.9.0
```

Reply:
I'd go with `requirements-dev.txt` and `requirements-dev.txt` without referencing each other, any dev will know how to deal with it. But I dont have strong opinion about it.

Reply:
I'd keep the original name `requirements.txt` instead of adding the `-prod` suffix there, as that's the most common name everyone looks for by default. Other than that it looks good to me, nice job.

Reply:
Done, thank you for the review 😄